packages/cx-link/doc/Link.md

```diff
@@ -1,37 +1,42 @@
-# CanvasX Link Component
+# CX Link Component
 
-CanvasX Link is based on [BodilessJS link](../../bodiless/Components/Link) component. It adds features and styles specific to CanvasX sites.
+The CX Link Component is based on the [BodilessJS Link Component](../../Components/Link/). While
+Bodiless Link is a generic Link component with tokens that can be combined however you choose, CX
+Link builds upon it, providing a sensible default combination of its generic tokens (i.e., features
+and styles), to help meet typical site-use expectations.
 
-- Provides an Editable Link component
-- Detects if the link is external and adds an appropriate icon automatically
-- Includes accessibility description
-- Uses Bodiless EditableLink under the hood
+- Provides an Editable Link component;
+- Detects if the link is external and adds an appropriate icon automatically;
```

Comment:
Detects if the link is to a downloadable  file and adds an appropriate icon automatically;